kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -282,9 +282,8 @@ int runInit() {
     return executionResult;
   }
 
-  // see HIVE-15820: comment at the head of beeline -e
-  @Override
-  boolean dispatch(String line) {
+  // see HIVE-15820: comment at the head of beeline -e only dispatch commands
+  boolean removeCommentsDispatch(String line) {
```

Comment:
what isthe difference

Reply:
The existing implementation also affects invocations in BeeLine
<img width="773" alt="image" src="https://github.com/apache/kyuubi/assets/26535726/f0643feb-5210-4f49-9689-0948539076b1">

Reply:
> what isthe difference
Remove comments in dispatch commands only execute command. 
for example：beeline -e '--   xxx' or -f script.sql remove comments，but not do anything for other case.